tests/ios/app/FW/FW.m

```diff
@@ -3,4 +3,8 @@
 
 @implementation FW
 
+- (void)foo {
+    NSLog(@"foo");
```

Comment:
It's probably not a big deal but this will print `foo` . Maybe add something more descriptive like `NSLog(@"Running %s", __PRETTY_FUNCTION__)`.